src/signals/settings/categories/Detail/components/CategoryForm/CategoryForm.tsx

```diff
@@ -139,7 +162,42 @@ const CategoryForm: FunctionComponent<CategoryFormProps> = ({
             ) : null}
 
             <FieldGroup>
-              <Label>Afhandeltermijn</Label>
+              <StyledDefinitionTerm>
+                <strong>Openbaar tonen</strong>
+              </StyledDefinitionTerm>
```

Comment:
Every `dt` should have a corresponding `dd`.

Reply:
I don't see the element in the DOM

Reply:
maybe you can look in the code
<img width="447" alt="Schermafbeelding 2022-03-08 om 11 11 02" src="https://user-images.githubusercontent.com/7254238/157215547-8a500b50-e3e0-4bca-8189-8d6f31751866.png">

Reply:
I have replaced the headings